packages/web-react/src/components/Container/README.md

```diff
@@ -0,0 +1,3 @@
+# Container
+
+Container docs.
```

Comment:
I think we should start adding a meaning to these `README`s, this is not very helpful 🙂. Possible options:

- add some helpful content like example usage, link to Storybook, and/or props table,
- remove all React component `README`s altogether.

Either way, it should not be a pain for us—Supernova is on it's way so we should ideally put here something we will be able to reuse further down the road.

Reply:
I would like to start adding links to the Storybook but I am waiting for Supernova. Storybook is now for me a visual testing tool, as the examples. However I have added basic usage information. So it has meaning now. I want to do it better in near future.